scripts/remove

```diff
@@ -64,6 +64,17 @@ ynh_remove_fpm_config
 
 ynh_remove_systemd_config
 
+#=================================================
+# REMOVE SERVICE FROM ADMIN PANEL
+#=================================================
+
+# Remove a service from the admin panel, added by `yunohost service add`
+if yunohost service status | grep -q $app
```

Comment:
This is ok, but: https://github.com/YunoHost/example_ynh/pull/73 looks better.

Reply:
Sure...
I let you upgrade every app as soon as it will be merged ;)

Reply:
If I approve your PR I don't have to?